### PR TITLE
update sdk-for-net

### PR DESCRIPTION
**Pull request by darshanhs90**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Graph.RBAC/Graph.RBAC/Generated/SdkInfo_GraphRbacManagementClient.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  internal static partial class SdkInfo
+  {
+      public static IEnumerable<Tuple<string, string, string>> ApiInfo_GraphRbacManagementClient
+      {
+          get
+          {
+              return new Tuple<string, string, string>[]
+              {
+                new Tuple<string, string, string>("GraphRbacManagementClient", "Applications", "1.6"),
```
Comment (dsgouda) [inline src/SDKs/Graph.RBAC/Graph.RBAC/Generated/SdkInfo_GraphRbacManagementClient.cs]: Please run `msbuild build.proj` and re-run the code generation script

File: src/SDKs/Graph.RBAC/Graph.RBAC/Generated/GroupsOperationsExtensions.cs [hunk 2]
```diff
@@ -132,37 +136,6 @@ public static async Task AddMemberAsync(this IGroupsOperations operations, strin
                 (await operations.AddMemberWithHttpMessagesAsync(groupObjectId, parameters, null, cancellationToken).ConfigureAwait(false)).Dispose();
             }
 
-            /// <summary>
```
Comment (dsgouda) [inline src/SDKs/Graph.RBAC/Graph.RBAC/Generated/GroupsOperationsExtensions.cs]: These are breaking changes. Please bump the minor version [here](https://github.com/Azure/azure-sdk-for-net/blob/b4aee1f3b3058845d0a2cec84628956ef8a4007a/src/SDKs/Graph.RBAC/Graph.RBAC/Microsoft.Azure.Graph.RBAC.csproj#L4)
Also update `PackageReleaseNotes` and `AssemblyInfo.cs`
If the REST API version is stable the nuget package version generated MUST also be stable
Please link the REST spec PR

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left a few comments

**Comment by shahabhijeet** [comment]
@darshanhs90 this PR will be closed end of this week as it's been a while this PR has been addressed.

**Pull request closed without merging**